src/backend/src/utils/reimbursement-requests.utils.ts

```diff
@@ -233,3 +233,12 @@ export const isAuthUserHeadOfFinance = (user: Prisma.UserGetPayload<typeof authU
 const isTeamIdInList = (teamId: string, teamsList: Team[]) => {
   return teamsList.map((team) => team.teamId).includes(teamId);
 };
+
+export const validateUserEditReimbursementRequest = async (user: User, reimbursementRequest: Reimbursement_Request) => {
```

Comment:
can you rename this to validateUserEditRRPermissions and then add a comment about what it's doing, like the comment for createNewProducts